salt/modules/inspectlib/kiwiproc.py

```diff
@@ -72,7 +72,7 @@ def export(self, name):
         self._set_packages(root)
 
         return '\n'.join([line for line in minidom.parseString(
-            etree.tostring(root, encoding='UTF-8', pretty_print=True)).toprettyxml(indent="  ").split("\n")
```

Comment:
This parameter is not recognized by the built-in `ElementTree` -- you have to have `lxml` installed or it will throw an exception here.